sdk/formrecognizer/tests.yml

```diff
@@ -8,5 +8,5 @@ extends:
     CloudConfig:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-        Location: 'canadacentral'
+        Location: 'centraluseuap'
```

Comment:
@kinelski @kristapratico can we change the cloud title from `Public` to `Canary` to make it clear that we're testing canary /euap regions and not prod regions?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/25359